tests/specs/3way/breakoutRooms.spec.ts

```diff
@@ -473,3 +475,12 @@ describe('BreakoutRooms', () => {
         await checkSubject(p2, myNewRoomName);
     });
 });
+
+
+/**
+ * Hangs up all participants (p1, p2 and p3)
+ * @returns {Promise<void>}
+ */
+function hangupAllParticipants() {
```

Comment:
Maybe we should move this in partisipants.ts so we can use it as util?

Reply:
Potentially we can do this, if we decide to keep this commit :)